modules/workflows/OrcaCopyToGlacierWorkflow/orca_copy_to_glacier_workflow.asl.json

```diff
@@ -50,9 +50,9 @@
             "granules": "{$.meta.processed_granules}",
             "buckets": "{$.meta.buckets}",
             "provider": "{$.meta.provider}",
-            "collection": "{$.meta.collection}",
             "files_config": "{$.meta.collection.files}",

```

Comment:
Wondering if we shouldn't remove the following since they are not needed.
```
           "granules": "{$.meta.processed_granules}",
            "buckets": "{$.meta.buckets}",
            "provider": "{$.meta.provider}",
            "collection": "{$.meta.collection}",
            "files_config": "{$.meta.collection.files}",
```

Reply:
Removed